lib/formats/json/json.flow

```diff
@@ -97,6 +97,14 @@ export {
 	// The setter to the nested field, dual to getJsonNestedField
 	setJsonNestedField(json: Json, fields: [string], value : Json) -> Json;
 
+	// Access to a nested field like obj.f1.f2.f3, where fields = [f1, f2, f3] including JsonArray indexes
+	getJsonNestedObjectOrArrayField(json: Json, fields: [string]) -> Json;
```

Comment:
The Json standard allows numbers to be used as object field names. Which makes the expected function result not entirely clear.
I would move this function to the app repository and remove the unused one (setJsonNestedObjectOrArrayField).